py/newsletter.py

```diff
@@ -24,13 +25,19 @@
 # create the document body
 with doc:
     # enclose all of the elements within div elements (currently set to ef-styling)
-    with div(cls='wrapper').add(div(cls='container')).add(table(cls='table outer')):
-        _head()
-        _greetings(greet_title)
-        _bodylineleft()
-        _bodylineright()
-        _bodynoline()
-        _foot()
+    with div(cls='wrapper').add(div(cls='container')).add(table(cls='outer height')):
```

Comment:
How come you needed to change this to how it is now?

Reply:
Before, all the sections were under the table tag but there were no rows or cells being created.

Reply:
I see. If you look at my head and greetings I actually create the rows and cells in there but I'll take your changes into consideration.

Reply:
Oh okay, sounds good. The way I thought about it was that it's probably better to create the rows and cells with the table itself.